resources/views/dropdown.blade.php

```diff
@@ -35,7 +36,7 @@ class="flex items-center focus:outline-none p-3 dropdown-button transition-defau
         x-transition:leave="transition ease-in duration-75"
         x-transition:leave-start="transform opacity-100 scale-100"
         x-transition:leave-end="transform opacity-0 scale-95"
-        class="origin-top-right absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses ?? 'w-40' }}"
+        class="origin-top-right absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses ?? 'w-40' }} {{ $fullScreen ? 'w-screen -mx-8 md:w-auto md:mx-0' : '' }}"
```

Comment:
I know, we can concatenate them but in my opinion it's clearer having them separated.